springfield/cms/templates/pattern-library/components/conditional-display/conditional-display.yaml

```diff
@@ -7,26 +7,52 @@ context:
         <p>No condition class is applied to this content.</p>
     - name: Firefox Users
       condition: is-firefox
+      html_class:
+      body_class: is-firefox
```

Comment:
Yay for adding to the pattern library as we go. 

They're all showing up for me though, so maybe the CSS is missing?